OverlayPlugin.Core/resources/opcodes.jsonc

```diff
@@ -3,42 +3,42 @@
 // For more information, see comments on https://github.com/OverlayPlugin/OverlayPlugin/issues/235
 {
     "Chinese": {
-        "2024.09.17.0000.0000": {
-            // Version: 7.0
+        "2024.10.10.0000.0000": {
+            // Version: 7.01
             "MapEffect": {
                 "opcode": 401,
```

Comment:
Did `MapEffect` opcode not actually change?

Reply:
> Did `MapEffect` opcode not actually change?

Yes, it's strange to me too, but it really hasn't changed.

Reply:
I guess there was always that 1/65536 chance that a given opcode wouldn't change... 0.0015%...